packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Anchor
+description: The anchor can be used to hyperlink to another URL. It accepts both internal and external URLs.
+---
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+<p>Example use cases may include:</p>
+
+<ul>
+  <li>Linking to another page within an app or website</li>
+  <li>Linking to an external domain outside of the current app or website</li>
+</ul>
+
+<p>Anchors can only include text.</p>
+
+</content>
+
+***
+
+<content>
+
+### When to Use an Anchor
+Use an anchor when you need to hyperlink to another page, app, or website.
+
+### Anchor vs. Button
+
+Anchors should be used in place of a button if you only need to create a hyperlink to some other page or content. Anchors should not be used for submitting a form, closing a modal, moving to the next step in a flow, or any other click action that a button should handle.
+
+If you need a click handler, you can use our button component. See the button documentation here.
+
+### Accessibility
+
+The anchor is built with standard accessible practices in mind. Those include an href attribute, link context, and opening and closing tags.
```

Comment:
I think the typings of Anchor is actually wrong now, because `React.ReactNode` accepts undefined but for A11y Anchor should always have children.  Fixing that should make it have the opening/close tag pairing.

Reply:
Do I need to do anything for the 0.1 launch on this?

Reply:
Nah but a Jira ticket would be useful

Reply:
Ticket created.